### PR TITLE
docs: inventory from csv

### PR DESCRIPTION
**Pull request by ajasnosz**
# Description

Adding documentation about updating inventory from csv file.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

N/A

## How Has This Been Tested?

N/A

## Checklist

- [x] My commit message is [conventional](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [x] I have run pre-commit on all files before creating the PR
- [x] I have made corresponding changes to the documentation
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: docs/configuration/poller-configuration.md [hunk 1]
```diff
@@ -148,3 +148,20 @@ microk8s helm3 upgrade --install snmp -f values.yaml splunk-connect-for-snmp/spl
 
 NOTE: If you decide to change the frequency of the profile without changing the inventory data, the change will be reflected after 
 next the walk process for the host. The walk happens every `walk_interval`, or on any change in inventory.
+
+#### Upgrade with csv file
+
+There is possibility to update inventory by making changes outside of `values.yaml`. It can be put to separate csv file and upgraded with passing `--set-file poller.inventory=<path_to_file>`.
```
Comment (omrozowicz-splunk) [inline docs/configuration/poller-configuration.md:154]: a possibility

Comment (omrozowicz-splunk) [reply docs/configuration/poller-configuration.md:154]: outside of the values.yaml

Comment (omrozowicz-splunk) [reply docs/configuration/poller-configuration.md:154]: upgraded passing

File: docs/configuration/poller-configuration.md [hunk 2]
```diff
@@ -148,3 +148,20 @@ microk8s helm3 upgrade --install snmp -f values.yaml splunk-connect-for-snmp/spl
...
+
+Example of CSV file configuration:
```
Comment (omrozowicz-splunk) [inline docs/configuration/poller-configuration.md:156]: the CSV file

File: docs/configuration/poller-configuration.md [hunk 3]
```diff
@@ -148,3 +148,20 @@ microk8s helm3 upgrade --install snmp -f values.yaml splunk-connect-for-snmp/spl
...
+
+```csv
+address,port,version,community,secret,security_engine,walk_interval,profiles,smart_profiles,delete
+10.202.4.202,,2c,public,,,3000,my_profile,,
+```
+
+Example of upgrade command with csv file:
```
Comment (omrozowicz-splunk) [inline docs/configuration/poller-configuration.md:163]: with the csv file

**Review by omrozowicz-splunk — CHANGES_REQUESTED** [review]

File: docs/configuration/poller-configuration.md (hunk 1 above)
Comment (ajasnosz) [reply docs/configuration/poller-configuration.md]: fixed

File: docs/configuration/poller-configuration.md (hunk 2 above)
Comment (ajasnosz) [reply docs/configuration/poller-configuration.md]: fixed

File: docs/configuration/poller-configuration.md (hunk 3 above)
Comment (ajasnosz) [reply docs/configuration/poller-configuration.md]: fixed

**Pull request merged**